ts/Series/Polygon/PolygonSeries.ts

```diff
@@ -10,60 +10,25 @@
 
 'use strict';
 
-import type ColorType from '../Core/Color/ColorType';
-import type ScatterPoint from './Scatter/ScatterPoint';
-import type ScatterPointOptions from './Scatter/ScatterPointOptions';
-import type ScatterSeries from './Scatter/ScatterSeries';
-import type ScatterSeriesOptions from './Scatter/ScatterSeriesOptions';
-import type { SeriesStatesOptions } from '../Core/Series/SeriesOptions';
-import type SVGPath from '../Core/Renderer/SVG/SVGPath';
-import BaseSeries from '../Core/Series/Series.js';
+import type PolygonPoint from './PolygonPoint';
+import type PolygonSeriesOptions from './PolygonSeriesOptions';
+import type SVGPath from '../../Core/Renderer/SVG/SVGPath';
+import BaseSeries from '../../Core/Series/Series.js';
 const { seriesTypes } = BaseSeries;
-import H from '../Core/Globals.js';
+import H from '../../Core/Globals.js';
 const { noop } = H;
-import LegendSymbolMixin from '../Mixins/LegendSymbol.js';
-import LineSeries from './Line/LineSeries.js';
+import LegendSymbolMixin from '../../Mixins/LegendSymbol.js';
+import LineSeries from '../Line/LineSeries.js';
+import ScatterSeries from '../Scatter/ScatterSeries.js';
```

Comment:
ScatterSeries should be deconstructed over BaseSeries.seriesTypes. Also remove the anonymous import of ScatterSeries.